python3/vimspector/debug_session.py

```diff
@@ -1975,8 +1980,16 @@ def PathsToAllConfigFiles( vimspector_base, current_file, filetypes ):
                                 os.path.dirname( current_file ) )
 
 
-def _SelectProcess():
-  custom_picker = settings.Get( 'custom_process_picker_expr' )
+def _SelectProcess( *args ):
+  custom_picker = settings.Get( 'custom_process_picker_func' )
   if custom_picker:
-    return int( vim.eval( custom_picker ) )
-  return int( utils.AskForInput( 'Enter Process ID: ' ) )
+    try:
+      return int( utils.Call( custom_picker, *args ) )
+    except vim.error:
+      pass
```

Comment:
Why? If I understand correctly, this falls back on the normal "enter the pid by hand" behavior, but why should we do it? If the pid pickers is broken, it is broken. The user goes to its vimrc and fixes it, no?